manifests/plugin/openscap.pp

```diff
@@ -6,4 +6,10 @@
 #
 class foreman::plugin::openscap {
   foreman::plugin {'openscap': }
+
+  if $foreman::cli::manage_root_config {
```

Comment:
Isn't the presence of this variable (and hammer_plugin_prefix, below) dependent on the parse order? I don't think it's reliable code. I think it probably needs to be in its own class that's included directly, not here.